src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -17,14 +17,13 @@
 import yaml
 import requests
 import urllib.request
-import signal
 from _thread import interrupt_main
 from psutil import process_iter, NoSuchProcess, AccessDenied, ZombieProcess, net_connections
 from knack.util import CLIError
 from knack.log import get_logger
 from knack.prompting import prompt_y_n
 from knack.prompting import NoTTYException
-from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core.commands.client_factory import get_mgmt_service_client, get_subscription_id
```

Comment:
Are we doing anything specific with mgmt_service client in troubleshoot? Lets remove if unused.